app/controllers/publishers_controller.rb

```diff
@@ -140,6 +141,26 @@ def update_unverified
     end
   end
 
+  def domain_status
```

Comment:
we should consider moving this action to its own Publishers API controller to prevent cluttering of the PublishersController.

i would also be satisfied with a TODO: comment